FtcRobotController/src/main/java/org/firstinspires/ftc/robotcontroller/external/samples/RobotAutoDriveToAprilTagOmni.java

```diff
@@ -34,6 +34,7 @@
 import com.qualcomm.robotcore.eventloop.opmode.TeleOp;
 import com.qualcomm.robotcore.hardware.DcMotor;
 import com.qualcomm.robotcore.util.Range;
+
```

Comment:
https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcTEufB5F2YJRjAvMZWorBw65u6y0WDe3fTEDGHTyccNog&s

can you get rid of this one too